frontend/src/helpers/index.ts

```diff
@@ -204,9 +204,35 @@ const defaultMaxNotificationsStored = 200;
 type NotificationStoreOptions = {
   max?: number;
 };
+
 function storeNotifications(notifications: Notification[], options: NotificationStoreOptions = {}) {
   const { max = defaultMaxNotificationsStored } = options;
-  localStorage.setItem('notifications', JSON.stringify(notifications.slice(0, max)));
+  const throttledSetItem: (notifications: Notification[]) => void = throttle(
+    (notifications: Notification[]) => {
+      localStorage.setItem('notifications', JSON.stringify(notifications.slice(0, max)));
+    },
+    5000
+  );
+  throttledSetItem(notifications);
```

Comment:
I think this will only apply the items after 5s but we should rather delay it only when there's been a call less than 5s before. i.e. A 1st call, and any calls that happen > 5s since the previous one should be done immediately.

Reply:
This PR doesn't work on the throttling part now, that is something i would like to handle in a different PR